clients/web/src/rest.js

```diff
@@ -171,6 +172,12 @@ let restRequest = function (opts) {
         throw new Error('restRequest requires a "url" argument');
     }
     args.url = `${getApiRoot()}${args.url.substring(0, 1) === '/' ? '' : '/'}${args.url}`;
+    // jQuery's ajax method prefers `method` over `type`.  Since we specify a
+    // default `method`, if the caller supplies a `type`, use it instead.
+    if (args.type) {
```

Comment:
The intent is fine -- we can remove the default of `method`/`type` being `GET`, since it is the default of `$.ajax` (but I'll add a comment to that effect).  Though we still need to leave the default `dataType` as `json`, as removing that is a breaking change.